packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,95 @@ path: /catalog/input-controls/form-fields/
 
 # Form Fields
 
-MDC Form Field provides an `mdc-form-field` helper class for easily making theme-aware, RTL-aware
-form field + label combos. It also provides an `MDCFormField` class for easily making input ripples
-respond to label events.
+MDC Form Field aligns a form field (for example, a checkbox) with its label, and makes it RTL-aware and [theme](../mdc-theme)-aware.
```

Comment:
What does form-field do with regards to mdc-theme? I see it applying typography styles but not theme...